godot-core/src/builtin/macros.rs

```diff
@@ -68,10 +68,10 @@ macro_rules! impl_builtin_traits_inner {
         impl Eq for $Type {}
     };
 
-    ( PartialOrd for $Type:ty => $gd_method:ident ) => {
```

Comment:
Nice improvement!

Reply:
Thanks! Clippy actually flagged the whole `PartialOrd` implementation as "incorrect" because there was an implementation for `Ord` which was interesting.